.github/workflows/docker-compose-elasticsearch-v7.yml

```diff
@@ -0,0 +1,10 @@
+version: '3.8'
```

Comment:
we don't need any new workflows, all permutations are achieved with the matrix strategy in the main workflow